### ISSUE TITLE
The user is unable to select the "see all synced tabs" button via voiceover

### ISSUE THREAD

**Issue by SimonBasca** [mozilla-mobile/firefox-ios#11736]
## Steps to reproduce
1. Enable voiceOver
2. Have a synced tab from desktop in the Jump back in section.
3. Try to select the "see all synced tabs" buttons
### Expected behavior
- The "see all synced tabs" buttons should be selectable
### Actual behavior
- The user is unable to select the "see all synced tabs" button via voiceover
### Device & build information
* Device: ?
* Build version: 59568c9ea028c11959f8b02501c6503372385047
### Notes
Attachments:

┆Issue is synchronized with this [Jira Task](https://mozilla-hub.atlassian.net/browse/FXIOS-4833)

**Comment by SimonBasca** [comment]
Closing the issue. While tapping on the synced tabs, Voiceover will inform that custom actions are available by swiping up/down. After swiping up/down the user is able to open tabs tray
